### PR TITLE
Fix problems with eval

### PR DESCRIPTION
**Pull request by SemyonBevzuk**
Added `get_nms_from_type` function to define function by nms type to replace `nms_op = eval(nms_type)`.

### CODE REVIEW

File: mmdet/ops/nms/nms_wrapper.py [hunk 1]
```diff
@@ -103,6 +103,21 @@ def nms(boxes, scores, iou_threshold, score_thr=0.0, max_num=-1, offset=0):
...
+    """A function to define an NMS operation by NMS type.
+
+    Arguments:
+        nms_type (str): string with nms type (e.g. 'nms').
+
+    Returns:
+        function: NMS function or 'None' if the nms_type is not supported.
+    """
+    nms_op = None
+    if nms_type is 'nms':
```
Comment (LeonidBeynenson) [inline mmdet/ops/nms/nms_wrapper.py]: Please, make check `==` instead of `is`
I'm not sure that the `is` will always work.

**Review by LeonidBeynenson — CHANGES_REQUESTED** [review]

File: mmdet/ops/nms/nms_wrapper.py (hunk 1 above)
Comment (SemyonBevzuk) [reply mmdet/ops/nms/nms_wrapper.py]: Done.

**Review by LeonidBeynenson — APPROVED** [review]
I approve the changes. LGTM!

**Comment by Ilya-Krylov** [comment]
@SemyonBevzuk please make a PR to training_extensions with this update

**Pull request merged**